napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -72,11 +72,17 @@ def __init__(self, parent: QWidget, axis: int):
         sep.setObjectName('slice_label_sep')
 
         settings = get_settings()
-
         self._fps = settings.application.playback_fps
+        settings.application.events.playback_fps.connect(
+            lambda e: setattr(self, 'fps', e.value)
+        )
+
         self._minframe = None
         self._maxframe = None
         self._loop_mode = settings.application.playback_mode
+        settings.application.events.playback_fps.connect(
```

Comment:
```suggestion
        settings.application.events.playback_mode.connect(
```

Maybe ? probably not `_fps` though.

Reply:
`playback_mode` seem to work when using  #3201 and corresponding IPython fix.

Reply:
thank you!  will have a look soon

Reply:
thanks for catching this